src/client/app.css

```diff
@@ -0,0 +1,14 @@
+@import url("https://fonts.googleapis.com/css?family=Orbitron&display=swap");
+@import url("https://fonts.googleapis.com/css?family=Khand|Orbitron&display=swap");
+
+@import "./helpers/variables.css";
+
+* {
```

Comment:
This can be added to body?

Reply:
is that bad practic?because in that case it ruins styles of main page

Reply:
It is a complex selector that can affect performance